tf/tfprocess.py

```diff
@@ -521,10 +543,10 @@ def construct_net(self, planes):
         self.weights.append(W_fc2)
         self.weights.append(b_fc2)
         h_fc2 = tf.nn.relu(tf.add(tf.matmul(h_conv_val_flat, W_fc2), b_fc2))
```

Comment:
I am wondering if this change should be accompanied by an increase in fc2 outputs to 256.  256 seems a small improvement for the old value head, but I wouldn't be surprised if its a stronger improvement for the new WDL head.